src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -610,5 +612,36 @@ export class NewNotebookAction extends Action {
 		}
 		return this.commandService.executeCommand(NewNotebookAction.INTERNAL_NEW_NOTEBOOK_CMD_ID, { connectionProfile: connProfile });
 	}
+}
+
+export class NotebookFindNext implements IEditorAction {
```

Comment:
>NotebookFindNext [](start = 13, length = 16)

nit: Would be clearer if you called these NotebookFindNextAction and NotebookFindPreviousAction